panther_analysis_tool/main.py

```diff
@@ -118,6 +123,13 @@
     "Tags",
 ]
 
+AVAILABLE_DESTINATION_NAMES = ("ExampleDestinationName",)
+
+OUTPUT_DISPLAY_NAMES = {
+    name: FakeDestination(destination_id=str(uuid4()), destination_display_name=name)
+    for name in AVAILABLE_DESTINATION_NAMES
+}
```

Comment:
Might be nice to add a comment describing why we are using these. It took me a few minutes to piece together why this was needed (rule.run(), destination dataclass, this...), but maybe that's just me... 🤷🏻‍♀️

Reply:
Yes, you are right, I'll explain it here briefly.